dashboard-backend/README.md

```diff
@@ -0,0 +1,5 @@
+Create a file `.env` and add the below credentials in that file.
```

Comment:
Can we provide a little more description of this, describing what, why, and how?

Reply:
@uditgaurav Added detailed installation steps